cli/src/main.rs

```diff
@@ -594,10 +667,66 @@ async fn try_deploy(
     return Err(DeploymentError::ServerError);
 }
 
+async fn release() {
+    if let Err(e) = try_release().await {
+        error!("{}", e);
+
+        std::process::exit(-1);
+    }
+}
+
+async fn try_release() -> Result<(), ReleaseError> {
```

Comment:
linking to what this is invoking for later
https://github.com/MyceliaNetwork/mycelia/pull/22/files#diff-9c5e61698d6d8a8ef4a3da7075c31857769134a932ac141d4eb62f68f131aa35R303